src/main/java/org/damap/base/rest/InternalStorageTranslationResource.java

```diff
@@ -0,0 +1,106 @@
+package org.damap.base.rest;
+
+import io.quarkus.security.Authenticated;
+import jakarta.annotation.security.RolesAllowed;
+import jakarta.inject.Inject;
+import jakarta.validation.Valid;
+import jakarta.ws.rs.*;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.UriInfo;
+import java.util.List;
+import lombok.extern.jbosslog.JBossLog;
+import org.damap.base.rest.base.resource.*;
+import org.damap.base.rest.storage.InternalStorageTranslationDO;
+import org.damap.base.rest.storage.InternalStorageTranslationService;
+
+@Path("/api/storages/translations")
+@Authenticated
+@Produces(MediaType.APPLICATION_JSON)
+@JBossLog
+public class InternalStorageTranslationResource
```

Comment:
Makes sense, I thought about it as well, but due to our Resource Interfaces (ResourceCreate<T>) this would require the same interface to be implemented for different types.